layers/+tools/lsp/packages.el

```diff
@@ -28,6 +28,7 @@
     :defer t
     :config
     (progn
+      (spacemacs|hide-lighter lsp-mode)
```

Comment:
I guess this is bad merge, we need reenable the lighter one more time https://github.com/syl20bnr/spacemacs/pull/11283 :)

Reply:
D'oh - sorry. I rebased this set of PRs so many times I obviously got sloppy :S